src/main/java/com/twilio/oai/AbstractTwilioGoGenerator.java

```diff
@@ -113,6 +112,21 @@ public OperationsMap postProcessOperationsWithModels(final OperationsMap objs, L
         return objs;
     }
 
+    private Map<String, ModelsMap> filterOutEnumResults( Map<String, ModelsMap> results) {
+        HashMap <String, ModelsMap> resultMap = new HashMap<>();
+        results.forEach((key ,value) -> {
+            if (!key.contains("_enum_")) {
```

Comment:
Is there a ticket in the backlog to revisit this?

Reply:
Nope, We will create it tomorrow and we will also create other improvement tickets for preview and alternative approach to __enum__ name in OAI schema models